Source/santa-driver/SantaDecisionManager.cc

```diff
@@ -57,6 +57,9 @@ bool SantaDecisionManager::init() {
   client_pid_ = 0;
   root_fsid_ = 0;
 
+  // Setup file modification prefix filter.
+  filemod_prefix_filter_ = new SantaPrefixTree();
```

Comment:
I think you should add a reset call in ConnectClient() to avoid leaving stale prefixes.

Reply:
If you add a reset in ConnectClient you get 2 benefits: 1) the tests won't break if you don't unload before running them and don't have to add reset calls to the test, 2) You can remove the reset on santad startup and only use the reset call when the config changes from when santad first starts.

Reply:
Done.